content/1_General/USACO_FAQs.mdx

```diff
@@ -73,6 +73,28 @@ with 100% certainty, but one way to check that there isn't anything obviously
 wrong with your solution is to [stress test](/general/basic-debugging?lang=cpp#stress-testing)
 against a model solution.
 
+## Q: What programming language should I use for USACO?
+
+The most popular languages that USACO supports are C++, Java, and Python. In
+general, we recommend the following:
+
+- If you already know one or more of these languages, just use the one you are
+  most comfortable with.
+- If you don't know any of these languages, you might as well start with C++, as
+  C++ users generally don't need to worry as much about their solutions being a
+  constant factor too slow. Furthermore, most modules currently lack Java and
+  Python support.
```

Comment:
ngl i feel like we've come a long way in java/python support...

Reply:
There are still modules in bronze/silver lacking Python support, I think it's fine to leave as is